tests/test_delete_files_in_range.rs

```diff
@@ -167,3 +167,39 @@ fn test_delete_files_in_range_with_delete_range() {
     assert_eq!(it.key(), b"5");
     assert!(!it.next());
 }
+
+#[test]
+fn test_delete_files_in_ranges() {
+    let path = TempDir::new("_rust_rocksdb_test_delete_files_in_multi_ranges").expect("");
+    let path_str = path.path().to_str().unwrap();
+    let db = initial_data(path_str);
+
+    // Delete files in multiple overlapped ranges.
+    // File ["key0", "key2"], ["key3", "key5"] should have been deleted,
+    // but file ["key6", "key8"] should not be deleted because "key8" is exclusive.
+    let mut ranges = Vec::new();
+    ranges.push(Range::new(b"key0", b"key4"));
```

Comment:
how we can ensure the keys are in different SSTs?

Reply:
What keys do you mean?

Reply:
maybe these keys are all in level 0 in one SST?

Reply:
The SST files are constructed deliberately in `initial_data()`, so we will have exactly three files with keys ["key0", "key2"], ["key3", "key5"], ["key6", "key8"].